smac/facade/smac_ac_facade.py

```diff
@@ -412,24 +411,14 @@ def __init__(self,
                 "Either use initial_design or initial_configurations; but not both")
 
         init_design_def_kwargs = {
-            'tae_runner': tae_runner,
-            'scenario': scenario,
-            'stats': self.stats,
+            'cs': scenario.cs,
             'traj_logger': traj_logger,
-            'runhistory': runhistory,
             'rng': rng,
+            'ta_run_limit': scenario.ta_run_limit,
             'configs': initial_configurations,
-            'intensifier': intensifier,
-            'aggregate_func': aggregate_func,
             'n_configs_x_params': 0,
             'max_config_fracs': 0.0
             }
-        if isinstance(intensifier, SuccessiveHalving):
```

Comment:
What's the reason why this is no longer needed?

Reply:
This was done to allow `InitialDesign` to fill the required challengers for a full successive halving run in initial design. Now that we don't have to fill configurations, this part was unnecessary.